galen-core/src/main/java/com/galenframework/config/GalenProperty.java

```diff
@@ -18,6 +18,8 @@
 public enum GalenProperty {
     SCREENSHOT_AUTORESIZE("galen.screenshot.autoresize", "true"),
     SCREENSHOT_FULLPAGE("galen.browser.screenshots.fullPage", "false"),
+    SCREENSHOT_HEADLESS("galen.browser.headless", "false"),
```

Comment:
Could you please rename this enum value? As it has nothing to do with screenshot